### PR TITLE
[single-staking-longtermstaking-balanceof] strategy for long term vault balance

### PR DESCRIPTION
**Pull request by macaronchef**
Added new strategy for Long Term Staking Vault

### CODE REVIEW

**Review by ChaituVR — APPROVED** [review]

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hi @macaronchef you have add your strategy single-staking-longtermstaking-balanceof to src/strategies/index.ts

**Comment by macaronchef** [comment]
> Hi @macaronchef you have add your strategy single-staking-longtermstaking-balanceof to src/strategies/index.ts

Hi @ChaituVR I added strategy to index.ts

File: src/strategies/index.ts [hunk 1]
```diff
@@ -184,6 +184,7 @@ import * as flexaCapacityStaking from './flexa-capacity-staking';
 import * as sunriseGamingUniv2Lp from './sunrisegaming-univ2-lp';
 import * as sunriseGamingStaking from './sunrisegaming-staking';
 import * as singleStakingAutoCompoundBalanceOf from './single-staking-autocompound-balanceof';
+import * as singleStakingLongTermStakingBalanceOf from './single-staking-longterm-staking-balanceof';
```
Comment (ChaituVR) [inline src/strategies/index.ts:187]:
```suggestion
import * as singleStakingLongTermStakingBalanceOf from './ssingle-staking-longtermstaking-balanceof';
```

File: src/strategies/index.ts [hunk 2]
```diff
@@ -615,6 +616,7 @@ const strategies = {
   'sunrisegaming-univ2-lp': sunriseGamingUniv2Lp,
   'sunrisegaming-staking': sunriseGamingStaking,
   'single-staking-autocompound-balanceof': singleStakingAutoCompoundBalanceOf,
+  'single-staking-longterm-staking-balanceof': singleStakingLongTermStakingBalanceOf,
```
Comment (ChaituVR) [inline src/strategies/index.ts:619]:
```suggestion
  'single-staking-longtermstaking-balanceof': singleStakingLongTermStakingBalanceOf,
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/index.ts [hunk 3]
```diff
@@ -184,6 +184,7 @@ import * as flexaCapacityStaking from './flexa-capacity-staking';
 import * as sunriseGamingUniv2Lp from './sunrisegaming-univ2-lp';
 import * as sunriseGamingStaking from './sunrisegaming-staking';
 import * as singleStakingAutoCompoundBalanceOf from './single-staking-autocompound-balanceof';
+import * as singleStakingLongTermStakingBalanceOf from './ssingle-staking-longtermstaking-balanceof';
```
Comment (ChaituVR) [inline src/strategies/index.ts:187]:
```suggestion
import * as singleStakingLongTermStakingBalanceOf from './single-staking-longtermstaking-balanceof';
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Looks like test cases are failing , can take a look @macaronchef ?

**Comment by macaronchef** [comment]
Tests and other issues fixed. Please review again @ChaituVR Thanks

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**